src/Common/src/System/Net/Security/Unix/SafeDeleteNegoContext.cs

```diff
@@ -38,7 +53,8 @@ public SafeDeleteNegoContext(SafeFreeNegoCredentials credential, string targetNa
             Debug.Assert((null != credential), "Null credential in SafeDeleteNegoContext");
             try
             {
-                _targetName = SafeGssNameHandle.CreatePrincipal(targetName);
+                _targetNameKerberos = SafeGssNameHandle.CreateTarget(targetName, false);
+                _targetNameNtlm = SafeGssNameHandle.CreateTarget(targetName, true);
```

Comment:
Nit: can you name the false/true arguments so it's clear what they are?

Reply:
Fixed